fastly/block_fastly_service_director.go

```diff
@@ -130,45 +134,18 @@ func (h *DirectorServiceAttributeHandler) Create(_ context.Context, d *schema.Re
 	return nil
 }
 
+// Read refreshes the resource state.
 func (h *DirectorServiceAttributeHandler) Read(_ context.Context, d *schema.ResourceData, _ map[string]interface{}, serviceVersion int, conn *gofastly.Client) error {
 	log.Printf("[DEBUG] Refreshing Directors for (%s)", d.Id())
 	directorList, err := conn.ListDirectors(&gofastly.ListDirectorsInput{
```

Comment:
In earlier versions of go-fastly the `Director` type returned from the `ListDirectors` API call was missing a `Backends` field. This meant this `Read` function had to make an API call to get all the service backends and to figure out which ones were members of the Director.

The following change removes the API call for service backends.